pkgs/tools/backup/borg/default.nix

```diff
@@ -67,5 +67,8 @@ python3.pkgs.buildPythonApplication rec {
     license = licenses.bsd3;
     platforms = platforms.unix; # Darwin and FreeBSD mentioned on homepage
     maintainers = with maintainers; [ flokli dotlambda globin ];
+
+    # 1.1.12 is broken on Python 3.8 (https://github.com/borgbackup/borg/issues/5214)
```

Comment:
Should we wait till tomorrow for 1.1.13? https://github.com/borgbackup/borg/issues/5214#issuecomment-640077775

Reply:
We currently use Python 3.7 on unstable, so that shouldn't be a blocker.

Reply:
Oh well, I missed the authors recommendation. So yeah, let's wait.

Reply:
Updated to 1.1.13.